src/svg-to-beziers.js

```diff
@@ -1,33 +1,33 @@
-var normalise = require("./normalise-svg.js");
+import { normalisePath as normalise } from './normalise-svg';
```

Comment:
with the export made a default, this can just "stay" what it was:
```suggestion
import normalise from './normalise-svg';
```